app/views/response_sets/_form.html.erb

```diff
@@ -44,6 +45,12 @@
       </div>
     </div>
 
+    <div class="hidden">
+      <% if defined?(parent_id) %>
+        <%= f.hidden_field :parent_id, :value => parent_id %>
```

Comment:
I don't think this is needed.  Once the parent has been extended there should not be a need to capture the parent_id in a hidden field and return to the server.